cpp/src/io/parquet/reader_impl_helpers.hpp

```diff
@@ -202,9 +233,22 @@ class aggregate_reader_metadata {
 
   [[nodiscard]] auto get_num_row_groups() const { return num_row_groups; }
 
-  [[nodiscard]] auto const& get_schema(int schema_idx) const
+  /**
+   * @brief Extracts the schema_idx'th SchemaElement from the pfm_idx'th file
+   *
+   * @param schema_idx The index of the SchemaElement to be extracted.
+   * @param pfm_idx The index of the per_file_metadata to extract SchemaElement from, default = 0 if
```

Comment:
Is the default useful here? I would remove it for clarity/explicitness, perhaps.

Reply:
Added this to not break this function's usage at places where it doesn't make sense to provide `pfm_idx` as all schema trees are identical (our usual case) so we just extract from the zeroth tree. Though not a strong preference one way or the other.

Reply:
I see. This is fine.